docs/topics/js/js-ir-migration.md

```diff
@@ -214,4 +170,39 @@ kotlin {
         binaries.executable()
     }
 }
+```
+
+## Additional troubleshooting tips when working with the Kotlin/JS IR compiler
+
+These hints may help you when troubleshooting problems in your projects using the Kotlin/JS IR compiler.
+
+### Make boolean properties nullable in external interfaces
+
+**Issue**: when you call `toString` on a `Boolean` from an external interface, you're getting an error like `Uncaught TypeError: Cannot read properties of undefined (reading 'toString')`. JavaScript treats the `null` or `undefined` values of a boolean variable as `false`. If you rely on calling `toString` on a `Boolean` that may be `null` or `undefined` (for example when your code is called from JavaScript code you have no control over), be aware of this:
```

Comment:
it, also, could be fail in casts to `Boolean` and `Any` (note not-null!) either explicit and implicit. Implicit ones could be generated by compiler, e.g. when refine generic type in override  (bridges), like:
https://pl.kotl.in/-NnNLbtOQ
```kotlin
external interface Foo {
    val bar: Boolean
}

interface A<T> {
    fun foo(a: T) = a
}

class B : A<Boolean> {
    override fun foo(a: Boolean) = a
}


fun main() {
    try {
        val o: Foo = js("{}")
	println("1")
        val bool = o.bar// as Boolean
        println("2")
        val a = B() as A<Boolean>
        println("3")
        println(a.foo(bool))
        println("done")
    } catch (e: Throwable) {
        println("fail")
    }
}
```

Reply:
`Boolean` as type parameter doesn't look like typical case :)